<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowk</class>
 <widget class="QMainWindow" name="MainWindowk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Taller</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>611</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Event</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>111</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="ExcludeEvents">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Events</string>
    </property>
   </widget>
   <widget class="QWidget" name="Grupos" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>631</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="LDR_En">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Habilitar LDR</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="LCD_En">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Habilitar LCD</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Interrupt_En">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Habilitar Interrupciones</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Control_En">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Habilitar control</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="LDR_Group">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>LDR</string>
     </property>
     <widget class="QLCDNumber" name="ValueLDR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>141</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="LCD_Group">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>20</y>
       <width>131</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>LCD 16*2</string>
     </property>
     <widget class="QPlainTextEdit" name="TextoLCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhNone</set>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="maximumBlockCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="ShowLCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearLCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Interrupt_Group">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Sensor de movimiento</string>
     </property>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="undoRedoEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="markdown">
       <string/>
      </property>
      <property name="tabStopDistance">
       <double>5.000000000000000</double>
      </property>
      <property name="acceptRichText">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Control_Group">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>191</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Control de temperatura</string>
     </property>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Set</string>
      </property>
      <widget class="QSpinBox" name="SetPoint_spin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <number>230</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
      <widget class="QPushButton" name="SetTemp_btn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Temperatura Actual</string>
      </property>
      <widget class="QLCDNumber" name="ValueTemp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Mode::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Filled</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QCheckBox" name="ShowDisplayTemp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>190</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar en pantalla</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ShowDisplayInterrupt">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar en pantalla</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ShowDisplayLDR">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar en pantalla</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ShowDisplayLCD">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar en pantalla</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>222</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>135</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>135</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Seleccione Puerto COM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignVCenter">
        <widget class="QComboBox" name="Ports_btn">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Seleccione puerto COM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="Connect_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Conectar/Desconectar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Conectar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Reset_btn">
         <property name="toolTip">
          <string>Cargar puertos COM</string>
         </property>
         <property name="text">
          <string>Refrescar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
